ste/JobsAdmin.go

```diff
@@ -420,12 +420,16 @@ func (ja *jobsAdmin) BytesOverWire() int64 {
 	return ja.pacer.GetTotalTraffic()
 }
 
-func (ja *jobsAdmin) AddSuccessfulBytesInActiveFiles(n uint64) {
-	atomic.AddInt64(&ja.atomicSuccessfulBytesInActiveFiles, int64(n))
+func (ja *jobsAdmin) AddSuccessfulBytesInActiveFiles(n int64) {
+	atomic.AddInt64(&ja.atomicSuccessfulBytesInActiveFiles, n)
 }
 
 func (ja *jobsAdmin) SuccessfulBytesInActiveFiles() uint64 {
-	return uint64(atomic.LoadInt64(&ja.atomicSuccessfulBytesInActiveFiles))
+	n := atomic.LoadInt64(&ja.atomicSuccessfulBytesInActiveFiles)
+	if n < 0 {
+		n = 0 // should never happen, but would result in nasty over/underflow if it did
```

Comment:
TODO: With the supposed debug env var enabled, consider panicking instead to catch any other issues?

Reply:
Tempting, but given that we don't have that env var yet, and that any error would "only" relate to status reporting rather than data integrity, I propose to leave this as-is.